zeebe/protocol-impl/src/test/java/io/camunda/zeebe/protocol/impl/JsonSerializableToJsonTest.java

```diff
@@ -2484,6 +2486,31 @@ final var record =
         """
       },
       /////////////////////////////////////////////////////////////////////////////////////////////
+      //////////////////////////////////// AuthorizationRecord ////////////////////////////////////
```

Comment:
❌ Please also add the empty `AuthorizationRecord` case. This record has optional properties so we should verify that if we don't set those, the mapping works and contains the default values.